src/Engine/ProtoCore/Properties/Resources.en-US.resx

```diff
@@ -489,4 +489,364 @@
   <data name="ModuloByZero" xml:space="preserve">
     <value>Integer modulo by zero</value>
   </data>
+  <data name="addadd_expected" xml:space="preserve">
+    <value>"++" expected</value>
+  </data>
+  <data name="add_expected" xml:space="preserve">
+    <value>"+" expected</value>
+  </data>
+  <data name="andand_expected" xml:space="preserve">
+    <value>"&amp;&amp;" expected</value>
+  </data>
+  <data name="and_expected" xml:space="preserve">
+    <value>"&amp;" expected</value>
+  </data>
+  <data name="asterisk_expected" xml:space="preserve">
+    <value>"*" expected</value>
+  </data>
+  <data name="catch_expected" xml:space="preserve">
+    <value>"catch" expected</value>
+  </data>
+  <data name="char_expected" xml:space="preserve">
+    <value>char expected</value>
+  </data>
+  <data name="closebrace_expected" xml:space="preserve">
+    <value>"}" expected</value>
+  </data>
+  <data name="closebracket_expected" xml:space="preserve">
+    <value>closebracket expected</value>
+  </data>
+  <data name="closeparen_expected" xml:space="preserve">
+    <value>closeparen expected</value>
+  </data>
+  <data name="comma_expected" xml:space="preserve">
+    <value>"," expected</value>
+  </data>
+  <data name="curvedash_expected" xml:space="preserve">
+    <value>"~\" expected</value>
+  </data>
+  <data name="dashdash_expected" xml:space="preserve">
+    <value>"--" expected</value>
+  </data>
+  <data name="divider_expected" xml:space="preserve">
+    <value>"/" expected</value>
+  </data>
+  <data name="doublecolumn_expected" xml:space="preserve">
+    <value>":" expected</value>
+  </data>
+  <data name="endline_expected" xml:space="preserve">
+    <value>endline expected</value>
+  </data>
+  <data name="EOF_expected" xml:space="preserve">
+    <value>EOF expected</value>
+  </data>
+  <data name="equalmark_expected" xml:space="preserve">
+    <value>"=" expected</value>
+  </data>
+  <data name="equalright_expected" xml:space="preserve">
+    <value>"=&gt;" expected</value>
+  </data>
+  <data name="equal_expected" xml:space="preserve">
+    <value>equal expected</value>
+  </data>
+  <data name="error" xml:space="preserve">
+    <value>error</value>
+  </data>
+  <data name="float_expected" xml:space="preserve">
+    <value>float expected</value>
+  </data>
+  <data name="greaterequal_expected" xml:space="preserve">
+    <value>greaterequal expected</value>
+  </data>
+  <data name="greaterthan_expected" xml:space="preserve">
+    <value>greaterthan expected</value>
+  </data>
+  <data name="hax_expected" xml:space="preserve">
+    <value>"#" expected</value>
+  </data>
+  <data name="ident_expected" xml:space="preserve">
+    <value>ident expected</value>
+  </data>
+  <data name="invalid_Associative_AccessSpecifier" xml:space="preserve">
+    <value>invalid Associative_AccessSpecifier</value>
+  </data>
+  <data name="invalid_Associative_AddOp" xml:space="preserve">
+    <value>invalid_Associative_AddOp</value>
```

Comment:
These errors don't make any sense to users. @junmendoza, @lukechurch please review these error messages and provide your comments.